Sources/LiUIComponents/Utils/LiHelperFunctions.swift

```diff
@@ -14,23 +14,21 @@
 
 import UIKit
 public class LiHelperFunctions {
-    public static func timeSince(post: String?) -> String {
-        guard let post = post else {
+    public static func timeSince(post date: String?) -> String {
+        guard let date = date else {
             return ""
         }
         let formatter = DateFormatter()
         formatter.dateFormat = LiUIConstants.dateTimeUTCFormat
-        guard let localDate = formatter.date(from: post) else {
-            print("Date format failed", post)
+        guard let localDate = formatter.date(from: date) else {
+            print("Invalid date format found", date)
             return ""
         }
-        return LiHelperFunctions.timeAgoSinceDate(date: localDate, numericDates: true)
+        return LiHelperFunctions.timeSince(date: localDate)
     }
```

Comment:
Not really comfortable with the idea of silently consuming any errors (in input or format) and returning an empty string. Would prefer if an exception or error is thrown and we allow the caller to decide how to handle it. If an implementation is required where empty string must be returned consider a decorated method or take a default value as an argument.

```
    public static func timeSince(post date: String?, default: String) -> String {
        guard let date = date else {
            return default
        }
        let formatter = DateFormatter()
        formatter.dateFormat = LiUIConstants.dateTimeUTCFormat
        guard let localDate = formatter.date(from: date) else {
            print("Invalid date format found", date)
            return default
        }
        return LiHelperFunctions.timeSince(date: localDate)
    }
```

or

```
public static func safeTimeSince(post date: String?) -> String {
  do {
    return try timeSince(date: date)
  } catch {
    print(error)
    return ""; // or a default value 
  }
}
```